### PR TITLE
Feat/file image loader

### PR DESCRIPTION
**Pull request by Crash--**
See commit message

### CODE REVIEW

File: src/lib/flags.js [hunk 1]
```diff
@@ -30,4 +30,5 @@ const flagsList = () => {
   flag('drive.enable-encryption')
   flag('drive.dacc-files-size-by-slug')
   flag('drive.breadcrumb.showCompleteBreadcrumbOnPublicPage') // flagName should use kebab case
+  flag('drive.thumbnails-pdf.enabled')
```
Comment (trollepierre) [inline src/lib/flags.js:33]: Did you set it in Bender for Cozy Internal?

File: src/drive/web/modules/filelist/FileIcon.jsx [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
+import FileImageLoader from 'cozy-ui/transpiled/react/FileImageLoader'
 import styles from 'drive/styles/filelist.styl'
 
 const FileIcon = ({ file, size, isEncrypted }) => {
   const isImage = file.class === 'image'
   const isShortcut = file.class === 'shortcut'
-
-  if (isImage)
+  const shoudUseThumbnailsForPDF = flag('drive.thumbnails-pdf.enabled')
+  if (isImage || (shoudUseThumbnailsForPDF && file.class === 'pdf'))
```
Comment (trollepierre) [inline src/drive/web/modules/filelist/FileIcon.jsx:12]: I think this feature needs a little time to be tested. If you don't have it, tell me :-)

Comment (Crash--) [reply src/drive/web/modules/filelist/FileIcon.jsx:12]: What do you want to test here?

Comment (trollepierre) [reply src/drive/web/modules/filelist/FileIcon.jsx:12]:
```
import React from 'react'
import { render } from '@testing-library/react'
import FileIcon from './FileIcon'

jest.mock('cozy-flags', () => () => true)
jest.mock('cozy-ui/transpiled/react/FileImageLoader', () => () => (
  <div data-testid="FileImageLoader" />
))

describe('FileIcon', () => {
  it('should return file image loader when file is image', () => {
    // Given
    const file = { class: 'image' }

    // When
    const { getByTestId } = render(<FileIcon file={file} />)

    // Then
    expect(getByTestId('FileImageLoader')).toBeInTheDocument()
  })

  it('should return file image loader when file is pdf', () => {
    // Given
    const file = { class: 'pdf' }

    // When
    const { getByTestId } = render(<FileIcon file={file} />)

    // Then
    expect(getByTestId('FileImageLoader')).toBeInTheDocument()
  })
})
```

Comment (trollepierre) [reply src/drive/web/modules/filelist/FileIcon.jsx:12]: or https://github.com/cozy/cozy-drive/commit/9d449f4a9a78c177eb4a3a0d9f4336f3866bf86e

Comment (Crash--) [reply src/drive/web/modules/filelist/FileIcon.jsx:12]: Typically what I call "testing for testing sake" but hey you're in charge here now so I cherry-picked your commit.

**Review by trollepierre — APPROVED** [review]

**Pull request merged**